Userland/Libraries/LibWasm/AbstractMachine/AbstractMachine.h

```diff
@@ -114,11 +114,6 @@ class Value {
         }
     }
 
-    explicit Value(u128 raw_value)
```

Comment:
```suggestion
        $this_{\text{is an example}}$
```

$this too, \mathfrak{IS AN EXAMPLE}$